src/basic.js

```diff
@@ -299,8 +299,14 @@ function ForceHLAllObjects() {
  * Delay then Force Highlight objects
  *  This is used to wait for DOM to populate before filling in problem details in left panel.
  */
-function delayForceHLAllObjects() {
-	setTimeout(ForceHLAllObjects, 100);
+function delayForceHLAllObjects(e) {
+  const ldf = Wa.app.layout.model.attributes.loadingFeatures;
+  const ty = (typeof e == 'object') ? (e.type===undefined ? '#' : e.type) : '#';
+  if (ldf) {
+    //console.log('dlyFrcHL - ldfeat ' + ldf + ' ty ' + ty);
+    setTimeout(function () { delayForceHLAllObjects(e); },50);
+  }
+  else { ForceHLAllObjects(); }
```

Comment:
Should it be `setTimeout(ForceHLAllObjects, 100)` as it used to be? I didn't try this change yet, but timeouts were to improve UX...

Reply:
After zoom or move end, WME requests "Features" from the back end and these will load for a second or so. I added the check to see if the load has complete. We dont want to start scanning segments if they arent all loaded yet. So instead of a pure delay, we have a check for ready, and a shorter delay between checks if not ready. Either 50 or 100 would work.